data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/coordination/partition/ExportPartition.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds.coordination.partition;
+
+import org.opensearch.dataprepper.model.source.coordinator.SourcePartitionStoreItem;
+import org.opensearch.dataprepper.model.source.coordinator.enhanced.EnhancedSourcePartition;
+import org.opensearch.dataprepper.plugins.source.rds.coordination.state.ExportProgressState;
+
+import java.util.Optional;
+
+/**
+ * An ExportPartition represents an export job needs to be run for tables.
+ * Each export job has an export time associate with it.
+ * Each job maintains the state such as total files/records etc. independently.
+ * The source identifier contains keyword 'EXPORT'
+ */
+public class ExportPartition extends EnhancedSourcePartition<ExportProgressState> {
```

Comment:
Not blocking this PR. The source coordination partition classes seem to be common with docdb (not sure if Aurora will have similar classes repeated again). Would it be possible to refactor out the common classes or interfaces?

Reply:
Yeah, I think it's possible. I can take a stab at it in a later PR. There are indeed common parts between docdb, dynamodb and rds source that we can refractor out, including the partition classes and other things. That can benefit other integrations in the future.